### ISSUE TITLE
turn.js "Fixed" but not visible

### ISSUE THREAD

**Issue by pictureworks** [blasten/turn.js#447]
I would like my pages to _always_ exist in the DOM (so I'm using class "fixed") but I don't want them to be always visible to the user (which they are by default). Is there a way to have them always exist in the DOM but only the active page/s are visible? Basically, I don't want the JS to keep deleting and recreating the pages (even though it's better on performance). Any ideas, anyone?

**Comment by pictureworks** [comment]
I've emailed the support line twice and this was left over a month ago and still no response? Maybe I'm being unreasonable but I would have thought my $99 would have bought me at least a response. I feel very cheated.

**Comment by shipleyr** [comment]
I agree. I've got the same problem. The inside cover of my book is always visible when I give it a class of fixed but I want it there when the book is open to provide the background display around the rest of the book. 

I must say so far this has not been a simple library to get up and running with... mostly because the documentation is lacking in examples. Been fumbling my way forward with very little assistance.
